src/lib.rs

```diff
@@ -553,10 +553,7 @@ pub struct ParsedMail<'a> {
 }
 
 impl<'a> ParsedMail<'a> {
-    /// Get the body of the message as a Rust string. This function tries to
-    /// unapply the Content-Transfer-Encoding if there is one, and then converts
-    /// the result into a Rust UTF-8 string using the charset in the Content-Type
-    /// (or "us-ascii" if the charset was missing or not recognized).
+    /// Get the un-charset-decoded body of the message as a Rust Vec<u8>.
```

Comment:
Update this comment to mention that the returned value has been content-transfer decoded.